runtime/python/pose_estimation/pose_estimation_utils.py

```diff
@@ -0,0 +1,506 @@
+from pathlib import Path
+import numpy as np
+import cv2
+from PIL import Image
+from hailo_platform import HEF
+from zenlog import logging
+
+# Joint pairs used for drawing pose estimations
+JOINT_PAIRS = [
+    [0, 1], [1, 3], [0, 2], [2, 4],
+    [5, 6], [5, 7], [7, 9], [6, 8], [8, 10],
+    [5, 11], [6, 12], [11, 12],
+    [11, 13], [12, 14], [13, 15], [14, 16]
+]
+
+
+def data_type2dict(net: str, data_type: str) -> dict:
+    """
+    initiates a dictionary where the keys are layers names and values are the requested data type
+    
+    Args:
+        net(str) : a string containing a path to hef the file
+        data_type(str) : the requested data type (e.g 'FLOAT32', 'UINT8', or 'UINT16')
+    
+    Returns:
+        Dict: layer name to data type 
+    """
+    hef = HEF(net)
```

Comment:
maybe you can pass the hef instead of initializing it again? you can create a method to get it from the HailoAsyncInfer class

Reply:
this function is being used before an instance of HailoAsyncInfer is being created...
i can pass the hef using:
`    output_type_dict = output_data_type2dict(HEF(args.net), 'FLOAT32')
`

Or, perhaps we can create a HEF prior to that and use the it both to create the dict and to init the HailoAsyncInfer class. this way we won't construct a HEF twice.